setup.py

```diff
@@ -1,10 +1,22 @@
-from setuptools import setup, find_packages
+import io
+from glob import glob
+from os.path import basename, dirname, join, splitext
+
+from setuptools import find_packages, setup
 
 PROJECT_URL = 'https://github.com/melexis/warnings-plugin'
 VERSION = '0.0.2'
 
+
+def read(*names, **kwargs):
+    return io.open(
+        join(dirname(__file__), *names),
+        encoding=kwargs.get('encoding', 'utf8')
+    ).read()
+
+
 setup(
-    name='warnings-plugin',
+    name='warnings',
```

Comment:
I dont think so - because there is namespace below which before made it ci.warnings-plugin and that namespace is mlx now, with name warnings... I would assume.